formatter.go

```diff
@@ -0,0 +1,5 @@
+package gocognit
+
+type Formatter interface {
+	Write([]Stat) error
+}
```

Comment:
I'm thinking about the Golang idiom, do you think make sense if we follow them? see https://pkg.go.dev/io

Reader interface -> Read
Writer interface -> Write
Seeker interface -> Seek
Closer interface -> Close